File: result/translate/starchat/java2python/python/FIND_EXPRESSION_DUPLICATE_PARENTHESIS_NOT.py
def f_gold(s):
        stack = []
        for i in s:
            if i == ')':
                if not stack:
                    return False
                top = stack.pop()
                count = 0
                while top!= '(':
                    count += 1
                    top = stack.pop()
                if count < 1:
                    return True
            else:
                stack.append(i)
        return False

File: result/translate/starchat/java2cpp/c++/FIND_EXPRESSION_DUPLICATE_PARENTHESIS_NOT.cpp
bool f_gold ( string s ) {
        stack<char> st;
        for (auto &c : s) {
            if (c == ')') {
                if (st.empty() || st.top()!= '(') return false;
                st.pop();
            } else {
                st.push(c);
            }
        }
        return st.empty();
    }